src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.HydraAdapter/CommonHelpers.cs

```diff
@@ -25,7 +25,7 @@ public static string GetDateTimeStringForService(DateTime date)
             // we have to use english culture because our user might be running 
             // PS in another culture and our service can't understand it.
             DateTimeFormatInfo dateFormat = new CultureInfo("en-US").DateTimeFormat;
-            return date.ToString("yyyy-MM-dd hh:mm:ss tt", dateFormat);
+            return date.ToUniversalTime().ToString("yyyy-MM-dd hh:mm:ss tt", dateFormat);
```

Comment:
I have written some method like ConvertToUTCDateTime() in policyconversions.cs .. pls check if you can reuse/generalize it